pkg/apis/networking/types.go

```diff
@@ -77,6 +87,26 @@ type NetworkPolicyIngressRule struct {
 	From []NetworkPolicyPeer
 }
 
+// NetworkPolicyEgressRule describes a particular set of traffic that pods can egress to
+// matched by a NetworkPolicySpec's podSelector. The traffic must match both ports and to.
+type NetworkPolicyEgressRule struct {
+	// List of destination ports for outgoing traffic.
+	//Each item in this list is combined using a logical OR. If this field is
+	// empty or missing, this rule matches all ports (traffic not restricted by port).
+	// If this field is present and contains at least one item, then this rule does note
+	// constrain outgoing traffic.
```

Comment:
"If this field is present and contains at least one item, then this rule allows traffic only if the traffic matches at least one port in the list". (Just copied from NetworkPolicyIngressRule.Ports)